syz-manager/hub.go

```diff
@@ -131,6 +134,19 @@ func (hc *HubConnector) loop() {
 		if hub == nil && query >= maxAttempts && !doneOnce {
 			hc.mgr.hubIsUnreachable()
 		}
+		// We used to send corpus updates (added/removed elements) to the hub in each sync.
+		// But that produced too much churn since hub algorithm is O(N^2) (distributing everything
+		// to everybody), and lots of new inputs are later removed (either we can't reproduce coverage
+		// after restart, or inputs removed during corpus minimization). So now we don't send new inputs
+		// in each sync, instead we aim at sending corpus once after initial triage. This solves
```

Comment:
Is it sent once after the initial triage or actually also each time `sync()` fails? We do reset `hub` in that case.

Reply:
Yes, after failed syncs as well (I assume errors shouldn't normally happen).

Reply:
Probably no really bad consequences indeed.